demo/addons/fmod/fmod_config_window.gd

```diff
@@ -0,0 +1,127 @@
+	get_node("%RolloffValue").connect("value_changed", self, "update_rolloff_scale_label")
+	
+	load_file()
+
+func load_file():
+	var config = ConfigFile.new()
+	
+	var err = config.load(config_file_path)
+
+	if err != OK:
+		#Set Default Values
+		config.set_value("Init", "SRIndex", 1)
+		config.set_value("Init", "SampleRate", 44100)
+		config.set_value("Init", "LiveUpdate", false)
+		config.set_value("Init", "SpeakerMode", 3)
+		config.set_value("Init", "NumChannels", 16)
+		config.set_value("Init", "NumListeners", 2)
+		config.set_value("Init", "DSPBufferSize", 1048)
+		config.set_value("Init", "ShowDebug", false)
+		config.set_value("Init", "LoadFMODOnStart", true)
+		
+		config.set_value("Banks", "Location", "")
+		config.set_value("Banks", "Autoload", "Master\n")
+		
+		config.set_value("3D", "DoplerScale", 1.0)
+		config.set_value("3D", "DistanceFactor", 32.0)
+		config.set_value("3D", "RolloffScale", 1.0)
+		
+		config.save(config_file_path)
+
+	#Change input locations to show loaded values
+	sample_rate_selector.selected = config.get_value("Init", "SRIndex")
+	speaker_mode_selector.selected = config.get_value("Init", "SpeakerMode")
+	LU_check.pressed = config.get_value("Init", "LiveUpdate")
+	num_channels_box.value = config.get_value("Init", "NumChannels")
+	num_listeners_value.value = config.get_value("Init", "NumListeners")
+	dsp_buffer_size.value = config.get_value("Init", "DSPBufferSize")
+	debug_checker.pressed = config.get_value("Init", "ShowDebug")
+	load_fmod_on_start_check.pressed = config.get_value("Init", "LoadFMODOnStart")
+	
+	banks_file_path.text = config.get_value("Banks", "Location")
+	banks_to_load.text = config.get_value("Banks", "Autoload")
+	
+	dopler_scale_slider.value = config.get_value("3D", "DoplerScale")
+	dist_factor_value.value = config.get_value("3D", "DistanceFactor")
+	rolloff_value.value = config.get_value("3D", "RolloffScale")
+
+func apply_changes():
+	var config = ConfigFile.new()
+	config.set_value("Init", "SRIndex", sample_rate_selector.selected)
+	config.set_value("Init", "SampleRate", int(sample_rate_selector.text))
+	config.set_value("Init", "SpeakerMode", speaker_mode_selector.selected)
+	config.set_value("Init", "LiveUpdate", LU_check.pressed)
+	config.set_value("Init", "NumChannels", int(num_channels_box.value))
+	config.set_value("Init", "NumListeners", int(num_listeners_value.value))
+	config.set_value("Init", "DSPBufferSize", int(dsp_buffer_size.value))
+	config.set_value("Init", "ShowDebug", debug_checker.pressed)
+	config.set_value("Init", "LoadFMODOnStart", load_fmod_on_start_check.pressed)
+	
+	config.set_value("Banks", "Location", banks_file_path.text)
+	config.set_value("Banks", "Autoload", banks_to_load.text)
+	
+	config.set_value("3D", "DoplerScale", dopler_scale_slider.value)
+	config.set_value("3D", "DistanceFactor", dist_factor_value.value)
+	config.set_value("3D", "RolloffScale", rolloff_value.value)
+	
+	config.save(config_file_path)
+
+func load_bank_names():
```

Comment:
If I set a wrong path, the bank autoload fails and return everything in the root of the project directory.

<img width="303" alt="image" src="https://user-images.githubusercontent.com/31470327/204178874-99f247b0-92e4-49b5-9be0-941ed53af999.png">